portfolio/src/main/webapp/mapStyles.js

```diff
@@ -0,0 +1,163 @@
+const mapStyles = [
```

Comment:
Is this the recommended way to set style values for the Maps API? It seems a bit strange to have values that look like css stored in a js file.

Reply:
thing is, styles are set using json, and have to be called when calling the constructor, so I think that to avoid having all of the clutter in scripts.js, having it on a separate file works better

Reply:
Ah gotcha, makes sense!

Reply:
thanks!